usr/share/rear/verify/CDM/default/410_use_replica_cdm_cluster_cert.sh

```diff
@@ -24,13 +24,24 @@ while true; do
     PrintError "Please enter a non-empty CDM cluster IP."
 done
 
+# The name of the tar file that is being downloaded has changed in Rubrik CDM v5.1.
+# Before Rubrik CDM v5.1 it was rubrik-agent-sunos5.10.sparc.tar.gz
+# since Rubrik CDM v5.1 it is rubrik-agent-solaris.sparc.tar.gz
+# cf. https://github.com/rear/rear/issues/2441
+
 CDM_SUNOS_TAR=rubrik-agent-sunos5.10.sparc.tar.gz
+CDM_SOLARIS_TAR=rubrik-agent-solaris.sparc.tar.gz
+CDM_TAR_FILE=$CDM_SUNOS_TAR
 cd /tmp
```

Comment:
Please use `$TMP_DIR` for temporary files

Reply:
In general `cd /some/dir` in a ReaR script is problematic.
Usually it should be `pushd /some/dir` with matching `popd`.

Regarding hardcoded `/tmp` I did
https://github.com/rear/rear/issues/2460

Reply:
At other places we have code like
```
  ( cd /some/dir
    some commands
  )
```
which uses a subshell
but I think normally code like
```
    pushd /seome/dir
    some commands
    popd
```
looks somehow "better" to me.

I wonder if we should perhaps do in the `Source` function
something like
```
function Source () {
    local current_dir=$( pwd )
    ...
    source "$source_file"
    source_return_code=$?
    cd $current_dir
```
to ensure that after each script
we are back in ReaR's usual working directory?

Reply:
I did https://github.com/rear/rear/issues/2461